src/fluree/db/dbfunctions/core.cljc

```diff
@@ -1,5 +1,6 @@
 (ns fluree.db.dbfunctions.core
-  (:require [clojure.tools.reader.edn :as edn]
+  (:require #?(:clj  [clojure.tools.reader.edn :as edn])
```

Comment:
if you `:refer [read-string]` here, then you would be able to get rid of the reader conditional below where you use this function in both platforms

Reply:
excluded clojure.core read-string & consolidated read-string logic.